metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -384,11 +387,21 @@ def run(self) -> None:
                 except SystemExit:
                     raise
                 except Exception as e:
-                    # TODO: Transformer errors should cause the pipeline to fail.
```

Comment:
this try catch is specific to transformers

there's an outer one that applies for the whole pipeline